authenticator/views.py

```diff
@@ -1,19 +1,74 @@
 #Django
 from django.shortcuts import render
-from django.contrib.auth import authenticate, login
-from django.contrib.auth.backends import BaseBackend
+from django.contrib.auth import authenticate
+from django.contrib.auth.models import User
 
+#Local model
+from authenticator.models import Personal_data
+
+
+#User creation in admin 
+def user_creation(request):
+
+    if request.method == 'POST':
+
+        username = request.POST['username']
+        password = request.POST['password']
+        password_conf = request.POST['password_conf']
+
+        if password != password_conf:
+            return f'Error'
+        else:
+            return f'Password accepted'
+
+        user = authenticate(username=username, password=password)
+        if user:
+            user = User.objects.create_user(username=username, password=password)
+        else: 
+            f'Django Authentication Error'
+
+
+#In another template, the completion of user data 
+def Complete_data(request, user): 
```

Comment:
if you completely define the user object inside the function, is not necessary to receive it here. You could only receive the request and define the user from scratch inside.

```
def Complete_data(request)
  user = {}
...
```